src/coinify/trade.js

```diff
@@ -246,12 +253,9 @@ CoinifyTrade.prototype.btcExpected = function () {
       return Promise.resolve(this.outAmountExpected);
     }
 
-    var fifteenMinutesAgo = new Date(new Date().getTime() - 15 * 60 * 1000);
     var oneMinuteAgo = new Date(new Date().getTime() - 15 * 60 * 1000);
```

Comment:
Good catch, that would have caused the expected BTC rate to only update once every 15 minutes.